src/service/jwt.service.js

```diff
@@ -0,0 +1,46 @@
+import 'dotenv/config';
+import jwt from 'jsonwebtoken';
+import { ConsoleLoger } from '../untils/consoleLoger.js';
+
+function sign(user) {
+  const token = jwt.sign(user, process.env.JWT_KEY, {
+    expiresIn: '15m',
+  });
+
+  return token;
+}
+
+function verify(token) {
+  try {
+    return jwt.verify(token, process.env.JWT_KEY);
+  } catch (error) {
+    ConsoleLoger.error(error);
```

Comment:
There is a typo in 'ConsoleLoger'. It should be 'ConsoleLogger'. Ensure that the import statement and all usages are corrected.